### DISCUSSION TITLE
etcd watch delay

### DISCUSSION THREAD

**Discussion by kellyseeme** [etcd-io/etcd#15757] [Q&A]
when i use i client to watch etcd key,and more events happen,the event will be dealy for 10min,how this happen?
in nomal case ,there is no question.
etcd 3.5.4

**Comment by jmhbnz** [comment]
Hey @kellyseeme - Thanks for raising this discussion. Do you mean that the event watch will be delayed? 

If so I will guess that you might be impacted by the watch stream starvation issue: https://github.com/etcd-io/etcd/issues/15402

**Reply by kellyseeme** [reply]
we have not use the tls settings,just for normal connection,

**Comment by serathius** [comment]
In correctly working etcd cluster watch delay should be around of 10ms.
